### PR TITLE
add weekly update March 20th

### PR DESCRIPTION
**Pull request by julianduque**

### CODE REVIEW

**Comment by yosuke-furukawa** [comment]
:+1:

File: weekly-updates/weekly-update.2015-03-20.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+* **net**: allow `server.listen()` to accept a `String` option for `port`, e.g. `{ port: "1234" }`, to match the same option being accepted in `net.connect()` as of [joyent/node#9268](https://github.com/joyent/node/pull/9268) (Ben Noordhuis) [#1116](https://github.com/iojs/io.js/pull/1116).
+* **tls**: further work on the reported memory leak although there appears to be a minor leak remaining for the use-case in question, track progress at [#1075](https://github.com/iojs/io.js/issues/1075).
+* **v8**: backport a fix for an integer overflow when `--max_old_space_size` values above `4096` are used (Ben Noordhuis) [#1166](https://github.com/iojs/io.js/pull/1166).
+* **platforms**: the io.js CI system now reports passes on **FreeBSD** and **SmartOS** (_Solaris_).
+* **npm**: upgrade npm to 2.7.1. See [npm CHANGELOG.md](https://github.com/npm/npm/blob/master/CHANGELOG.md#v271-2015-03-05) for details. Summary:
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-03-20.md]: According to 03-06.md, weekly updates does not write npm change summary.
see https://github.com/iojs/evangelism/pull/37/files

File: weekly-updates/weekly-update.2015-03-20.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  * [`6823807`](https://github.com/npm/npm/commit/6823807bba) [#7121](https://github.com/npm/npm/issues/7121) `npm install --save` for Git dependencies saves the URL passed in, instead of the temporary directory used to clone the remote repo. Fixes using Git dependencies when shrinkwwapping. In the process, rewrote the Git dependency caching code. Again. No more single-letter variable names, and a much clearer workflow. ([@othiym23](https://github.com/othiym23))
```
Comment (e-jigsaw) [inline weekly-updates/weekly-update.2015-03-20.md]: shrinkwwapping -> shrinkwapping ?

Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-03-20.md]: Thanks for your report.

hm....
https://github.com/npm/npm/blob/master/CHANGELOG.md#gitsanity

File: weekly-updates/weekly-update.2015-03-20.md (hunk 1 above)
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-03-20.md]: https://github.com/iojs/evangelism/issues/16#issuecomment-77664152

**Pull request merged**